src/test/java/com/bobocode/blyznytsia/bibernate/util/CaseUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package com.bobocode.blyznytsia.bibernate.util;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class CaseUtilTest {
+  @Test
```

Comment:
I would prefer to use Parametrized test

Reply:
Good idea. Currently, I'm heavily refactoring test cases, and I will include this change with the next PR

Reply:
Ok